teams/tikv/team.json

```diff
@@ -52,15 +52,21 @@
     "reviewers": [
         "Fullstop000",
         "Jibbow",
+        "MrCroxx",
```

Comment:
I didn't find the promotion rule for the new team architecture. Based on existing rule for engine SIG, @MrCroxx is not yet qualified for reviewer position.
```suggestion
```

Reply:
I do find it a bit tiresome to manually give interns triage permission in certain repos where the review request isn't managed by bot. But it is still manageable for now.

Reply:
Which repos is involved in this case? Why do you have to give someone triage permission while they are not reviewer?

Be aware that PingCAP doesn't take a special role in TiKV community and we are supposed to introduce concept like "A PingCAP Intern".

Reply:
`/cc` won't work in many repos, e.g. tikv/rocksdb,tikv/rust-rocksdb,tikv/raft-engine, in which case interns without triage permission can't request review on their own.

Reply:
@tabokie Yep. And the moment a reviewer or committer should help in the way and the person who is not a reviewer can use `cc @reviewer` as a basic mention.

Many projects do not depend on this conventions and if reviewers active on those repo the case should be eased.

A problem is, if someone is not "A PingCAP Intern", he/she suffers the same problem, why "A PingCAP Intern" so special?

Reply:
Of course another way think of it is we can enable auto cc mechanism. But any is not the reason we setup a special role for PingCAP Intern in TiKV community. A good case is, an intern as a general community member, acts well and deserves his/her reviewer role.

Reply:
Agreed. A special role would be overkill.